--- cv/include/cv.h.original	2008-07-28 08:39:20.000000000 +0900
+++ cv/include/cv.h	2008-07-28 08:40:37.000000000 +0900
@@ -1056,18 +1056,18 @@
 struct CvFeatureTree;
 
 /* Constructs kd-tree from set of feature descriptors */
-CVAPI(CvFeatureTree*) cvCreateFeatureTree(CvMat* desc);
+CVAPI(struct CvFeatureTree*) cvCreateFeatureTree(CvMat* desc);
 
 /* Release kd-tree */
-CVAPI(void) cvReleaseFeatureTree(CvFeatureTree* tr);
+CVAPI(void) cvReleaseFeatureTree(struct CvFeatureTree* tr);
 
 /* Searches kd-tree for k nearest neighbors of given reference points,
    searching at most emax leaves. */
-CVAPI(void) cvFindFeatures(CvFeatureTree* tr, CvMat* desc,
-		    CvMat* results, CvMat* dist, int k = 2, int emax = 20);
+CVAPI(void) cvFindFeatures(struct CvFeatureTree* tr, CvMat* desc,
+		    CvMat* results, CvMat* dist, int k CV_DEFAULT(2), int emax CV_DEFAULT(20));
 
 /* Search kd-tree for all points that are inlier to given rect region. */
-CVAPI(int) cvFindFeaturesBoxed(CvFeatureTree* tr,
+CVAPI(int) cvFindFeaturesBoxed(struct CvFeatureTree* tr,
 		    CvMat* bounds_min, CvMat* bounds_max,
 		    CvMat* results);
 
